go.mod

```diff
@@ -6,18 +6,23 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/containers/image/v5 v5.4.3
 	github.com/containers/storage v1.19.0
+	github.com/docker/distribution v2.7.1+incompatible
 	github.com/docker/docker v1.4.2-0.20191219165747-a9416c67da9f
 	github.com/docker/go-units v0.4.0
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
+	github.com/google/go-cmp v0.4.0 // indirect
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
 	github.com/opencontainers/runc v1.0.0-rc9
 	github.com/opencontainers/runtime-spec v0.1.2-0.20190618234442-a950415649c7 // indirect
 	github.com/opencontainers/selinux v1.5.1
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.4.2
+	github.com/sirupsen/logrus v1.5.0
```

Comment:
1.5.0 is the broken version that we don’t want to use. Either keep this as is (AFAICS it has nothing to do with this PR), or update to 1.6.0.